crates/aggchain-proof-core/src/bridge/mod.rs

```diff
@@ -670,20 +670,6 @@ mod tests {
             (prev, new)
         };
 
-        // 1. Get the prev inserted GER hash chain (previous block on L2)
```

Comment:
Nit: Since the point number one was removed, the subsequent comments should be renumbered (or just have the numbering removed).